### PR TITLE
Shadid

### PR DESCRIPTION
**Pull request by saranatour1**
Latest changes to cart, about us, restaurants and restaurant page , and further edits

### CODE REVIEW

File: Feedme/src/main/webapp/WEB-INF/index.jsp
Comment (Yousef-labadi) [inline Feedme/src/main/webapp/WEB-INF/index.jsp:1]: good job shadid

**Review by mohadshadid — APPROVED** [review]
Continue

File: Feedme/src/main/webapp/WEB-INF/rest.jsp
Comment (Yousef-labadi) [inline Feedme/src/main/webapp/WEB-INF/rest.jsp:1]: i trust you

File: Feedme/src/main/webapp/WEB-INF/index.jsp
Comment (saranatour1) [reply Feedme/src/main/webapp/WEB-INF/index.jsp:1]: good work guys !

File: Feedme/src/main/webapp/WEB-INF/rest.jsp
Comment (saranatour1) [reply Feedme/src/main/webapp/WEB-INF/rest.jsp:1]: 🤣

**Review by Yousef-labadi — APPROVED** [review]

**Pull request merged**